Source/WebCore/platform/RectEdges.h

```diff
@@ -73,6 +73,9 @@ template<typename T> class RectEdges {
     void setBottom(const T& bottom) { setAt(BoxSide::Bottom, bottom); }
     void setLeft(const T& left) { setAt(BoxSide::Left, left); }
 
+    void flipX() { T stash = left(); left() = right(); right() = stash; }
```

Comment:
Rather than mutating the `RectEdges<>`, it would be nicer to have functions that return flipped copies.

Reply:
I could go either way, and kinda prefer this one tbh, but sure. :)